quesma/quesma/router_v2.go

```diff
@@ -354,3 +359,210 @@ func ConfigureSearchRouterV2(cfg *config.QuesmaConfiguration, dependencies quesm
 
 	return router
 }
+
```

Comment:
these functions mostly assemble the Elasticsearch API responses - I'd suggest moving them off `router_v2.go` to a separate file `es_api_responses.go`/ `es_responses.go` or even `responses.go` or something alike. WDYT?

Reply:
Done!